src/Service.Tests/Unittests/SqlMetadataProviderUnitTests.cs

```diff
@@ -178,9 +188,19 @@ private static async Task CheckExceptionForBadConnectionStringHelperAsync(string
             catch (DataApiBuilderException ex)
             {
                 // use contains to correctly cover db/user unique error messaging
-                Assert.IsTrue(ex.Message.Contains(DataApiBuilderException.CONNECTION_STRING_ERROR_MESSAGE));
-                Assert.AreEqual(HttpStatusCode.ServiceUnavailable, ex.StatusCode);
+                // if sw is not null it holds the error messaging
+                string error = sw is null ? ex.Message : sw.ToString();
+                Assert.IsTrue(error.Contains(DataApiBuilderException.CONNECTION_STRING_ERROR_MESSAGE));
                 Assert.AreEqual(DataApiBuilderException.SubStatusCodes.ErrorInInitialization, ex.SubStatusCode);
+                if (sw is not null)
```

Comment:
Check for the newly added post processing error message here in the exception to stdout?